fastjet/internal/delaunay/point.go

```diff
@@ -112,15 +112,38 @@ func (p *Point) findNearest() {
 }
 
 // inTriangle checks whether the point is in the triangle and whether it is on an edge.
-func (p *Point) inTriangle(t *Triangle) (inside, edge bool) {
+func (p *Point) inTriangle(t *Triangle) location {
 	o1 := predicates.Orientation(t.A.x, t.A.y, t.B.x, t.B.y, p.x, p.y)
 	o2 := predicates.Orientation(t.B.x, t.B.y, t.C.x, t.C.y, p.x, p.y)
 	o3 := predicates.Orientation(t.C.x, t.C.y, t.A.x, t.A.y, p.x, p.y)
 	if o1 == predicates.CCW && o2 == predicates.CCW && o3 == predicates.CCW {
-		// point is inside the triangle, but not on an edge
-		return true, false
+		return inside
 	}
-	outside := o1 == predicates.CW || o2 == predicates.CW || o3 == predicates.CW
-	// point is either outside or on an edge.
-	return !outside, !outside
+	if o1 == predicates.CW || o2 == predicates.CW || o3 == predicates.CW {
+		return outside
+	}
+	return onEdge
+}
+
+// location is the position of a point relative to a triangle
+type location int
+
+const (
+	inside location = iota
+	onEdge
+	outside
+)
+
+func (l location) String() string {
+	switch l {
+	case inside:
+		return "Inside Triangle"
```

Comment:
Mixed case in messages is a bit of a no-no. I'd simplify this to "inside", "edge" and "outside", then people can decide how they want to use them.